commands/records/create.js

```diff
@@ -126,9 +126,11 @@ function validateDataAgainstSchema (collectionConfig, data, user, callback) {
 
   const schemaValidated = righto(checkSchemaValidations, schema, data, user, errors);
 
-  const rulesPassed = righto(checkSchemaRules, rules, data, user, errors, righto.after(schemaValidated));
+  schemaValidated(function (error) {
+    if (error) { callback(error); }
 
-  rulesPassed(callback);
+    callback(null, data);
+  });
```

Comment:
This could be cleaner with a `righto.mate`:

```
const result = righto.mate(data, righto.after(schemaValidated));

result(callback);
```

Reply:
So much nicer!